backend/fms_core/models/sample.py

```diff
@@ -117,12 +117,19 @@ class Sample(TrackedModel):
     sample_kind = models.ForeignKey(SampleKind, on_delete=models.PROTECT,
                                     help_text="Biological material collected from study subject "
                                               "during the conduct of a genomic study project.")
-    name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH, validators=[name_validator],
-                            help_text="Sample name.")
+
+    name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH,
+                            validators=[name_validator],
+                            help_text="Sample name.",
+                            error_messages={'blank': 'Sample name cannot be blank'})
+
     alias = models.CharField(max_length=200, blank=True, help_text="Alternative sample name given by the "
                                                                    "collaborator or customer.")
-    individual = models.ForeignKey("Individual", on_delete=models.PROTECT, related_name="samples", help_text="Individual associated "
-                                                                                     "with the sample.")
+    individual = models.ForeignKey("Individual",
+                                    on_delete=models.PROTECT,
+                                    related_name="samples",
+                                    help_text="Individual associated with the sample.",
+                                    error_messages={'null': 'Individual could not be processed due to missing information. Taxon and ID are necessary.'})
```

Comment:
Same as below (my comment related to the container attribute)

Reply:
Same change as the container message. 
```suggestion
                                    error_messages={'null': 'Individual could not be processed due to missing information.'})
```

Reply:
Same as above it implies missing user submitted information... this could happen for other reasons (e.g. someone forgot to create an individual altogether inside the app.